io.github.Q1CHENL.fig.json

```diff
@@ -0,0 +1,63 @@
+{
+  "id": "io.github.Q1CHENL.fig",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "fig",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--filesystem=xdg-pictures:ro",
+    "--filesystem=xdg-download:ro"
```

Comment:
you had removed these now you have put them back. This is a gtk4 app. it should use portals by default.

Reply:
I removed all --filesystem-* access. But then the file chooser dialog cannot see any system files out of the sandbox.